PWGGA/GammaConv/AliCaloPhotonCuts.cxx

```diff
@@ -4814,11 +4814,14 @@ void AliCaloPhotonCuts::ApplyNonLinearity(AliVCluster* cluster, Int_t isMC)
           if(fClusterType==1){
             energy /= (FunctionNL_DPOW(energy, 1.1251817141, -0.8420328354, -0.0800008954, 0.9562653194, -0.6683378769, -0.1064755375));
             energy /= FunctionNL_kSDM(energy, 0.977706, -4.21058, -0.0938915) ;
-          }
-          if( (fCurrentMC==k17e2 || fCurrentMC==k16k5a) && fClusterType==3) {
+          } else if( (fCurrentMC==k17e2 || fCurrentMC==k16k5a) && fClusterType==3) {
```

Comment:
Hi Adrian,

could you change the logic in a similar way here as for line 4875ff? This would make it much more readable.

Regards,
Friederike

Reply:
Did you mean like this?